src/api.ts

```diff
@@ -55,33 +57,33 @@ export const extractCommitFromImage = ( imageName: string ): CommitHash => {
 
 export const extractEnvironmentFromImage = ( image: ContainerInfo ): RunEnv => {
 	return image.Labels.calypsoEnvironment || undefined;
-}
+};
 
 /**
  * Polls the local Docker daemon to
  * fetch an updated list of images
  */
 export async function refreshLocalImages() {
 	const images = await docker.listImages();
-	const isTag = ( tag: string ) => tag.startsWith( config.build.tagPrefix );
-	const hasTag = ( image: Docker.ImageInfo ) => image.RepoTags && image.RepoTags.some( isTag );
-
-	state.localImages = new Map(
-		images
-			.filter( hasTag )
-			.map( image => [ image.RepoTags.find( isTag ), image ] as [ string, Docker.ImageInfo ] )
-	);
+	state.localImages = new Set( images );
```

Comment:
`state.localImages` now holds a different data structure that includes images not built by dserve (i.e images where `tag.startsWith( config.build.tagPrefix );` is false).

I updated `getLocalImages()` to apply a equivalent filtering, and replaced all usages of `state.localImages`.